pandas/io/html.py

```diff
@@ -236,48 +241,58 @@ def _text_getter(self, obj):
         """
         raise AbstractMethodError(self)
 
-    def _parse_td(self, obj):
-        """Return the td elements from a row element.
+    def _equals_tag(self, obj, tag):
+        """Returns whether an individual DOM node matches a tag
 
         Parameters
         ----------
         obj : node-like
```

Comment:
Let's take advantage of this time to re-specify this as a proper data type if possible.